src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -147,29 +147,33 @@
               </a>
             </div>
             <div class="d-flex flex-column ms-4 me-4 mt-4 mb-4">
-              <ng-template #healthChecks>
-                <cd-health-checks *ngIf="healthData?.health?.checks"
-                                  [healthData]="healthData.health.checks"></cd-health-checks>
-              </ng-template>
-
               <div class="d-flex flex-row col-md-3 ms-4">
-                <i *ngIf="healthData.health?.status"
-                   [ngClass]="[healthData.health.status | healthIcon, icons.large2x]"
-                   [ngStyle]="healthData.health.status | healthColor"
-                   [title]="healthData.health.status"></i>
+                <i  *ngIf="healthData.health?.status"
```

Comment:
it looks like there is somehow an extra space here?
```suggestion
                <i *ngIf="healthData.health?.status"
```

Reply:
I have given this space to align all tags in one line

Reply:
same I have given space for aligning other tags below